### ISSUE TITLE
How to finetune the model to improve the accuracy of answering specific questions?

### ISSUE THREAD

**Issue by aixiaodewugege** [salesforce/LAVIS#289]
Thanks for your brilliant work!

In my own image question answering dataset, I only care about the accuracy of a few specific questions. What is the best way to fine-tune Blip2?
